test/controllers/test_ontologies_controller.rb

```diff
@@ -282,11 +282,63 @@ def test_detach_a_view
     assert_equal onto["viewOf"], ont.id.to_s
   end
 
+  def test_ontology_agents
+    ontologies_and_submissions = create_ontologies_and_submissions(ont_count: 2, submission_count: 1, process_submission: true)
+    submission1 = ontologies_and_submissions[2].first.submissions.last
+    submission2 = ontologies_and_submissions[2].last.submissions.last
+
+    ontology_acronym1 = ontologies_and_submissions[1].first
+    ontology_acronym2 = ontologies_and_submissions[1].last
+
+    submission1.bring(*OntologySubmission.agents_attrs)
+    submission2.bring(*OntologySubmission.agents_attrs)
+
+    # To insure that we don't have duplicated agents in the response
+    agent_syphax = _create_agent(name: 'Syphax', type: 'person')
+
+    submission1.publisher = [_create_agent(name: 'Bilel', type: 'person'), agent_syphax]
+    submission1.hasContributor = [_create_agent(name: 'Clement', type: 'person'), agent_syphax]
+
+    submission2.publisher = [_create_agent(name: 'Imad', type: 'person'), _create_agent(name: 'Serine', type: 'person')]
+
+    submission1.save
+    submission2.save
+
+
+    get "/ontologies/#{ontology_acronym1}/agents"
+
+    response = MultiJson.load(last_response.body)
+    assert_equal response.length, 3
+    expected_result = ['Bilel', 'Syphax', 'Clement']
+    response.each do |r|
+      assert expected_result.include?(r["name"])
```

Comment:
can be replaced  with `assert_includes r["name"], ['Bilel', 'Syphax', 'Clement']`